tomland.cabal

```diff
@@ -53,6 +53,7 @@ library
                      , text ^>= 1.2
                      , time >= 1.8 && < 1.10
                      , transformers ^>= 0.5
+                     , containers ^>= 0.5
```

Comment:
I think here versions should be `containers >= 0.5.7 && < 0.7`. Also, could you please put this dependency according to alphabetic order?